include/glow/Optimizer/GraphOptimizer/FunctionPasses.h

```diff
@@ -16,19 +16,28 @@
 #ifndef GLOW_OPTIMIZER_GRAPHOPTIMIZER_FUNCTIONPASSES_H
 #define GLOW_OPTIMIZER_GRAPHOPTIMIZER_FUNCTIONPASSES_H
 
+#include "glow/Optimizer/GraphOptimizer/CompilationContext.h"
 #include "glow/Optimizer/GraphOptimizer/FunctionPass.h"
+#include "glow/Optimizer/GraphOptimizer/Pipeline/Pipeline.h"
+#include "glow/Support/Support.h"
 
 namespace glow {
 
+/// Declare all FunctionPass classes.
```

Comment:
Should we have a version of FUN_PASS that takes a base class as a parameter, so that one could say e.g. `FUN_PASS(myPassName, FunctionPassFollowedByDCE)` or something like that?

Reply:
Also, if the property `needs to run DCE after the pass` is not inherent to a pass, but is rather a property defined by the pipeline (e.g. it uses iterate until fixed point), then it should be rather the pass manager who decides to call DCE. But it should probably be a Graph IR specific PassManager who does it as other pass managers (for low-level IR) won't even have the DCE pass.